### ISSUE TITLE
Error reported if global type is prefixed by `::`

### ISSUE THREAD

**Issue by timwoj** [bbannier/zeek-language-server#496]
Screenshot from emacs:

![Screenshot 2024-08-05 at 9 16 50 AM](https://github.com/user-attachments/assets/2ce29ef5-f3aa-4de8-a5c9-02ab8fba69c5)

That was fine when it read `GLOBAL::r`, but using `GLOBAL::` is deprecated in 7.0 and becoming an error in 7.1.

**Comment by bbannier** [comment]
This is an issue with the grammar, and e.g., `zeek-format` also fails with this new syntax,

```zeek
type r: record { };

event zeek_init()
	{
	local x: ::r;
	}
```

```console
$ zeek-format foo.zeek
cannot parse line 5, col 9: ": :"
type r: record { };

event zeek_init()
        {
        local x : : : r;
        }
```

**Comment by bbannier** [comment]
Fixed by d992f859414ffcc9b220731a51cd3deb16dd466e, part of zeek-language-server-0.50.0.
